lbz/events/api.py

```diff
@@ -29,7 +29,7 @@ def __init__(self) -> None:
 
     def __repr__(self) -> str:
         return (
-            f"<EventAPI bus: {self._bus_name} Events: pending={len(self._pending_events)} "
+            f"<EventAPI bus: {self.bus_name} Events: pending={len(self._pending_events)} "
```

Comment:
I don't think you need to use the public property when you are inside the class 😉